x-pack/plugin/otel-data/src/yamlRestTest/resources/rest-api-spec/test/30_aggregated_metrics_tests.yml

```diff
@@ -0,0 +1,193 @@
+  - match: { hits.hits.0.fields.metricset\.name: ["service_transaction"] }
+  - match: { hits.hits.0.fields.metricset\.interval: ["10m"] }
+  - do:
+      indices.get_data_stream:
+        name: metrics-service_transaction.10m.otel-default
+  - match: { data_streams.0.hidden: true }
+---
+"metrics-service_transaction.60m must be hidden":
+  - do:
+      bulk:
+        index: metrics-service_transaction.60m.otel-default
+        refresh: true
+        body:
+          - create: {}
+          - '{"@timestamp":"2024-07-18T14:48:33.467654000Z" ,"attributes":{"processor.event":"metric","transaction.root":false, "metricset.name" : "service_transaction"},"resource":{"attributes":{ "metricset.interval": "60m" } } }'
+  - is_false: errors
+  - do:
+      search:
+        index: metrics-service_transaction.60m.otel-default
+        body:
+          fields: ["metricset.name", "metricset.interval"]
+  - length: { hits.hits: 1 }
+  - match: { hits.hits.0.fields.metricset\.name: ["service_transaction"] }
+  - match: { hits.hits.0.fields.metricset\.interval: ["60m"] }
+  - do:
+      indices.get_data_stream:
+        name: metrics-service_transaction.60m.otel-default
+  - match: { data_streams.0.hidden: true }
+---
+"metrics-transaction.10m must be hidden":
+  - do:
+      bulk:
+        index: metrics-transaction.10m.otel-default
+        refresh: true
+        body:
+          - create: {}
+          - '{"@timestamp":"2024-07-18T14:48:33.467654000Z" ,"attributes":{"processor.event":"metric","transaction.root":false, "metricset.name" : "transaction"},"resource":{"attributes":{ "metricset.interval": "10m" } } }'
+  - is_false: errors
+  - do:
+      search:
+        index: metrics-transaction.10m.otel-default
+        body:
+          fields: ["metricset.name", "metricset.interval"]
+  - length: { hits.hits: 1 }
+  - match: { hits.hits.0.fields.metricset\.name: ["transaction"] }
+  - match: { hits.hits.0.fields.metricset\.interval: ["10m"] }
+  - do:
+      indices.get_data_stream:
+        name: metrics-transaction.10m.otel-default
+  - match: { data_streams.0.hidden: true }
+---
+"metrics-transaction.60m must be hidden":
+  - do:
+      bulk:
+        index: metrics-transaction.60m.otel-default
+        refresh: true
+        body:
+          - create: {}
+          - '{"@timestamp":"2024-07-18T14:48:33.467654000Z" ,"attributes":{"processor.event":"metric","transaction.root":false, "metricset.name" : "transaction"},"resource":{"attributes":{ "metricset.interval": "60m" } } }'
+  - is_false: errors
+  - do:
+      search:
+        index: metrics-transaction.60m.otel-default
+        body:
+          fields: ["metricset.name", "metricset.interval"]
+  - length: { hits.hits: 1 }
+  - match: { hits.hits.0.fields.metricset\.name: ["transaction"] }
+  - match: { hits.hits.0.fields.metricset\.interval: ["60m"] }
+  - do:
+      indices.get_data_stream:
+        name: metrics-transaction.60m.otel-default
+  - match: { data_streams.0.hidden: true }
+
```

Comment:
An additional generic test we could do here is to do a terms aggregation on metricset.interval. We'd expect only one bucket with the value "1m". When setting `expand_wildcards:open,hidden`, the aggregation is supposed to return all three values.

Reply:
👍 Added these tests for all 4 aggregation indices.